community/sccache/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+cargo fetch \
+    --locked \
```

Comment:
Probably don't need the `\` on this line

Reply:
oops, fixed